docs/dev-client-networking.md

```diff
@@ -628,16 +628,15 @@ client to delete, overwrite, or ignore descendant updates.
 
 Event handlers that queue messages for later processing can be found in
 `listenForNetworkMessages` defined in
-[`src/utils/listen-for-network-message.ts`](https://github.com/mozilla/hubs/blob/master/src/utils/listen-for-network-messages.ts).
+[`src/utils/listen-for-network-message.ts`](https://github.com/Hubs-Foundation/hubs/blob/master/src/utils/listen-for-network-messages.ts).
 
 
 ### Persisting Networked Entity State
 
 By default, `networked instanciated entities` which are created with
 the built-in [`createNetworkedEntity()`](#createnetworkedentity) function,
 are removed when their creator (client) is disconnects. In order to persist
-these entities the entity must be
-[`pinned`](https://hubs.mozilla.com/docs/hubs-features.html). Only
+these entities the entity must be pinned. Only

```

Comment:
I think this could just be changed to this?
```suggestion
these entities the entity must be [pinned](https://docs.hubsfoundation.org/hubs-features.html). Only

```